### DISCUSSION TITLE
Meccano / Erector parts creation?

### DISCUSSION THREAD

**Discussion by nilsschmidt1337** [nilsschmidt1337/ldparteditor#885] [Q&A] [answered]
@Symbian9 wrote:
> How to setup LDPartEditor for creating Meccano/Erector (imperial units) & Merkur (metric units) sets parts? (at least for strips & plates)
> REFERENCES
> 
>     https://en.wikipedia.org/wiki/Meccano
>     https://meccanoindex.co.uk/Drawings/Parts.php?id=1634405614
>     https://www.usmeccano.com/other/meccstrips.htm
>     https://en.wikipedia.org/wiki/Erector_Set
>     https://en.wikipedia.org/wiki/Merkur_(toy)

**Comment by nilsschmidt1337** [comment, marked as answer]
Hi @Symbian9,

yes, you could use the LDraw file format to model the parts in the unit of your choice (https://www.ldraw.org/article/218.html).
The format does not enforce you to use the LDraw Unit (LDU) for your measurements.
This will, however, make the parts from the LDraw library incompatible with the parts created by you. 

One skilled "part author" has already created "Vex IQ" parts with LDraw 
(https://philohome.com/vexldraw/vexldraw.htm).

Here is a simple "strip", which I created without taking the real measurements into account.

<img width="441" alt="A strip" src="https://user-images.githubusercontent.com/11047164/179250414-63037c36-284e-4940-81b3-71c68c85183f.png">

```
0 Strip  1 x  3
0 Name: strip-1x3.dat
0 Author: Nils Schmidt [BlackBrick89]

0 BFC CERTIFY CCW

1 16 40 0 0 0 0 15 0 1 0 -15 0 0 2-4disc.dat
1 16 40 10 0 0 0 15 0 -1 0 -15 0 0 2-4disc.dat
1 16 40 0 0 0 0 15 0 1 0 -15 0 0 2-4edge.dat
1 16 40 10 0 0 0 15 0 -1 0 -15 0 0 2-4edge.dat
1 16 40 0 0 0 0 15 0 10 0 -15 0 0 2-4cyli.dat
0 BFC INVERTNEXT
1 16 30 0 0 10 0 0 0 10 0 0 0 10 4-4cyli.dat
1 16 30 0 0 10 0 0 0 1 0 0 0 10 4-4ndis.dat
1 16 30 10 0 10 0 0 0 -1 0 0 0 10 4-4ndis.dat
1 16 30 0 0 10 0 0 0 1 0 0 0 10 4-4edge.dat
1 16 30 10 0 10 0 0 0 -1 0 0 0 10 4-4edge.dat

0 BFC INVERTNEXT
1 16 0 0 0 10 0 0 0 10 0 0 0 10 4-4cyli.dat
1 16 0 0 0 10 0 0 0 1 0 0 0 10 4-4ndis.dat
1 16 0 10 0 10 0 0 0 -1 0 0 0 10 4-4ndis.dat
1 16 0 0 0 10 0 0 0 1 0 0 0 10 4-4edge.dat
1 16 0 10 0 10 0 0 0 -1 0 0 0 10 4-4edge.dat

1 16 -40 0 0 0 0 -15 0 1 0 15 0 0 2-4disc.dat
1 16 -40 10 0 0 0 -15 0 -1 0 15 0 0 2-4disc.dat
1 16 -40 0 0 0 0 -15 0 1 0 15 0 0 2-4edge.dat
1 16 -40 10 0 0 0 -15 0 -1 0 15 0 0 2-4edge.dat
1 16 -40 0 0 0 0 -15 0 10 0 15 0 0 2-4cyli.dat
0 BFC INVERTNEXT
1 16 -30 0 0 10 0 0 0 10 0 0 0 10 4-4cyli.dat
1 16 -30 0 0 10 0 0 0 1 0 0 0 10 4-4ndis.dat
1 16 -30 10 0 10 0 0 0 -1 0 0 0 10 4-4ndis.dat
1 16 -30 0 0 10 0 0 0 1 0 0 0 10 4-4edge.dat
1 16 -30 10 0 10 0 0 0 -1 0 0 0 10 4-4edge.dat

4 16 -10 0 10 -20 0 10 -20 0 -10 -10 0 -10
4 16 20 0 10 10 0 10 10 0 -10 20 0 -10
4 16 -10 10 -10 -20 10 -10 -20 10 10 -10 10 10
4 16 20 10 -10 10 10 -10 10 10 10 20 10 10
4 16 40 10 10 -40 10 10 -40 10 15 40 10 15
4 16 40 10 -15 -40 10 -15 -40 10 -10 40 10 -10
4 16 40 0 -10 -40 0 -10 -40 0 -15 40 0 -15
4 16 40 0 15 -40 0 15 -40 0 10 40 0 10
2 24 -40 0 -15 40 0 -15
2 24 40 10 -15 -40 10 -15
2 24 40 0 15 -40 0 15
2 24 -40 10 15 40 10 15
4 16 40 0 15 40 10 15 -40 10 15 -40 0 15
4 16 -40 0 -15 -40 10 -15 40 10 -15 40 0 -15
```

**Reply by ghost** [reply]
Thanks!
